packages/builders/cliff.toml

```diff
@@ -29,8 +29,10 @@ body = """
 		  {% endif %}\
 			{{ commit.message | upper_first }} ([{{ commit.id | truncate(length=7, end="") }}]({{ self::remote_url() }}/commit/{{ commit.id }}))\
 		{% if commit.breaking %}\
-			{% for breakingChange in commit.footers %}\
```

Comment:
You could also use the `| filter(breaking=true)` filter or w/e it is

Reply:
I chose this way because it allows us to do other things with other footers easily if we wanted. Also reads a bit cleaner to me.